charts/cluster-api-runtime-extensions-nutanix/templates/ccm/nutanix/manifests/helm-addon-installation.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.hooks.clusterAutoscaler.helmAddonStrategy.defaultValueTemplateConfigMap.create }}
```

Comment:
```suggestion
{{- if .Values.hooks.ccm.helmAddonStrategy.defaultValueTemplateConfigMap.create }}
```

Reply:
Fixed in https://github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/pull/38/commits/b7f79b8fc3c51c1789900a4bf7be42fe5a9b1ccd